### ISSUE TITLE
If you change localization of site, screenshots should also change localization

### ISSUE THREAD

**Issue by rakleed** [linearmouse/linearmouse.org#18]
When changing the localization of the site, it would be nice if the screenshots to also change the localization.

**Comment by lujjjh** [comment]
I completely agree with you. I'm thinking about the best practice for assets localization.
